src/frontend/src/views/LandingPageV2/index.tsx

```diff
@@ -0,0 +1,50 @@
+import Head from "next/head";
+import { useRouter } from "next/router";
+import React, { useEffect, useState } from "react";
+import { ROUTES } from "src/common/routes";
+import { useUserInfo } from "../../common/queries/auth";
+import Footer from "./components/Footer";
+import Hero from "./components/Hero";
+import IntroSection from "./components/IntroSection";
+import PartnersSection from "./components/PartnersSection";
+import QuoteSlider from "./components/QuoteSlider";
+import UseCases from "./components/UseCases";
+import { Container, LoadingText } from "./style";
+
+export default function Homepage(): JSX.Element {
+  const { data: userInfo, isLoading } = useUserInfo();
+  const [isRedirecting, setIsRedirecting] = useState(false);
+  const router = useRouter();
+
+  /**
```

Comment:
It seems like maybe this comment was copied over. I'm wondering if we might want to remove attribution in that case.